integration-tests/test/shared/env.ts

```diff
@@ -106,4 +122,82 @@ export class OptimismEnv {
   ): Promise<CrossDomainMessagePair> {
     return waitForXDomainTransaction(this.watcher, tx, direction)
   }
+
+  async relayXDomainMessages(
```

Comment:
I feel like this should use the same thing that was built for etherscan?

Reply:
We only gave etherscan `getMessagesAndProofsForL2Transaction` but maybe we should be giving them this tool instead...